src/js/components/form-elements/v2/ButtonFileSelect.jsx

```diff
@@ -65,6 +65,9 @@ const ButtonFileSelect = ({
           if (validateFileType(files)) {
             onFileUpload(files);
           }
+          // After upload, we need to clear the value,
+          // due to the bug on Chrome that the same file can't be uploaded twice
+          e.target.value = null;
```

Comment:
There was a bug visible only on Chrome (it was working fine on Firefox) that you can't upload the same file twice. It seems to be a known bug, as I found one stack article about that: https://stackoverflow.com/questions/9155136/chrome-file-upload-bug-on-change-event-wont-be-executed-twice-with-the-same-fi